### ISSUE TITLE
change css file issue

### ISSUE THREAD

**Issue by seeden** [istarkov/babel-plugin-webpack-loaders#36]
Hi

I have two files index.js and index.css (index.js requires index.css)
When I change index.css it will use old version.
I need to change index.js too if I want to get latest noncached index.css.

**Comment by istarkov** [comment]
Looks like I need to add `BABEL_DISABLE_CACHE=1` into README.md (I use this env  variable anywhere in package.json)
Thank you.

**Comment by seeden** [comment]
I am using it too but I was wondering if there is any other option.

**Comment by istarkov** [comment]
I'll look in babel sources. Looks like there is no other options in documentation. Any help will be appreciated.

**Comment by istarkov** [comment]
Looks like babel plugin `babel-plugin-transform-inline-environment-variables` must have the same issue if env variable has chaged. For now I'll add warning here, and will create an issue at babel, to get a good solution from babel community

**Comment by seeden** [comment]
Thank you @istarkov . It sounds good

**Comment by istarkov** [comment]
https://phabricator.babeljs.io/T1186 It is already where, will read when come 2 home.

**Comment by istarkov** [comment]
PR #37,  now in npm version `0.3.6`

**Comment by thangngoc89** [comment]
Ouch. That's why I get this warning 

```
babel-plugin-webpack-loader:
To avoid caching errors you need to set BABEL_DISABLE_CACHE=1 environment variable.
More information at issue #36
```

**Comment by seeden** [comment]
I guess that they closed the ticket as resolved without any fix

**Comment by istarkov** [comment]
As plugins has no knowledge about caching, there is no way to pass additional cache key information about revalidation etc.
I've looked into code and cache cleared if `BABEL_ENV`, `NODE_ENV` variables and/or plugin options has changed. As I can't change this variables inside plugin, the only way now is to ask user to set `BABEL_DISABLE_CACHE=1` or to set undocumented option `cache=false`.
As `cache=false` undocumented, the only way is to ask users to set `BABEL_DISABLE_CACHE=1`

And I have zero idea what I can suggest to babel to make cache revalidation algorithm smart.

**Comment by seeden** [comment]
Yep. I understand. The interesting thing is that the Webpack is working with the babel(without BABEL_DISABLE_CACHE). Maybe they are able to force the babel somehow.

**Comment by istarkov** [comment]
There is no need for webpack to re-transpile js files if something has changed in css file. Because it uses something like `webpack_require` for includes. But here css-module definition in placed inside js.

**Comment by marcustisater** [comment]
Thanks for the update guys :+1: 

The problem is that it **slows** down our build script, it's understandable because it's disabling the cache. Any other workaround or follow up on this would be much appreciated.
